templates/web/bromley/reports/_list-filters.html

```diff
@@ -0,0 +1,32 @@
+[% IF use_section_wrapper %]
```

Comment:
Do we want to have a shared template between all these identical ones using this HTML?